docs/to_clarify.md

```diff
@@ -18,16 +18,26 @@
 
 1. Stated under the section "Possible Use Cases" that the solution should "begin their check-in journey before arrival and complete it upon arrival".
 
-    a. What needs to be done during the checking-in process?
+    a. What needs to be done during the digital checking-in process? Are there any particular details/ workflow where it must be adhered to provide a check-in process?
 
     b. Which part of the process can guests complete up to before arrival, and what else do they need to do to complete the check-in process upon arrival?
 
     c. On the same note, what needs to be done during the checking-out process?
 
+    d. I would like to confirm the current traditional workflow of the check-in process.(Please fill me in if i got the procedure wrong)
+    Guest - Provide personal particulars, Photos of Passport/Identity Card, Booking Documents.?
+    
+    Front Desk staffs - Receiving the guest and collect relevant documents from guest. Do verification and give out hotels access card. Guide Guest to their room.
+
+
 1. Stated under the section "What Are We Looking For" that the solution should "verify travel and identification documents, booking documents, payment details, and make payment upon check-out, integrated with ICA (or other government stipulated system) for authentication and verification".
 
     a. Which government stipulated system is currently being used, and does the existing system provide API for our solution to communicate with it?
 
     b. Can we have several different samples of the above documents and the details of what are being verified for these documents?
 
+    c. What is the current architecture: Microservices or Monolith? 
+
+    d. How does the systems communicate with each other: REST API, SOAP, gRPC?
```

Comment:
I think this segment shouldn't be parked under my question, but it should be part of the first question under `Forum`:

> What are the "existing solutions and databases" that we need to integrate with as mentioned in Point 2 of "Possible Use Cases"?

Let's remove

> What is the current architecture: Microservices or Monolith? 

I think it's not a relevant question for now. That's because I don't think we are expected to touch their code; we just have to integrate with them by calling the relevant API. So we don't need to be concerned with their architecture. Sorry for asking you to write this down in my previous comment.

> How does the systems communicate with each other: REST API, SOAP, gRPC?

Similarly for this, this question as it is isn't relevant. We should be asking how should our solution integrate with the existing solutions. Using REST, SOAP, gRPC or others?

Reply:
It's okay sir. If there is a need, we will put it back. Will remove and update.